### DISCUSSION TITLE
Is my movement monitored to send traffic suggestions for others?

### DISCUSSION THREAD

**Discussion by slrslr** [osmandapp/OsmAnd#12104] [General]
On app called Waze, i think that i have participated in traffic situation monitoring so the app could pick fastest route for everyone.
But in OsmAnd 3.9.10 i do not see the answer to this question and whether or to which level traffic situation is used to improve the routing.

**Comment by vshcherb** [comment]
No, OsmAnd doesn't collect any data from user devices
